### DISCUSSION TITLE
Real Experience Score woes

### DISCUSSION THREAD

**Discussion by Anoroc** [vercel/next.js#35824] [Help]
Howdy,

I'm trying to get a better grasp on why the contrast between logging my Real Experience Score vs Lighthouse or other lab data. I understand the differences between the two, However, I'm struggling to find insight in order to resolve scores with the former. 

With that being said, I just pushed an update to block SEMBot which has been bombarding my site. Can a scraper like semBot cause performance issues on a Nextjs build? (It's been about an hour since I blocked the bot, so no useful feedback yet). Perhaps a majority of our visitors are viewing on poor connections?

For some insight, my lighthouse score is 100s across the board where RES is hovering between 60 - 83. However, I've seen it drop as low as the mid 20s. 

Thanks!

**Comment by icyJoseph** [comment]
Real Experience Score is an interesting metric indeed, I've seen mine be stable at 100 for days, and then suddenly a 7. 

- Lighthouse is a tool that goes against a check list, and that takes various time/load measurements on the site. By this very fact, it is dependent of the configuration with which it is run. On one of my laptops, I have to dwarf the CPU to x8, while in the other, to x2.
- With Lighthouse you can sort of, try to setup your site for success, but it doesn't mean it will be successful in terms of performance.
- If your site is mostly pre-rendered, the next.js server is pretty much just passing down HTML + bundles, and that shouldn't be so hard on it, even if a bot is trying to scrap so much. Usually bots don't have a runtime either, so I guess very few calls are actually made to API routes. If you do have ISR or gSSP, then I'd expect some heavy-load from a scrapper bot.
- I would think that the geographic location of users, specially in mobile, does affect the site metrics a lot. 

However I'd also like to hear about your FCP, LCP, CLS, and FID (if any).

**Comment by Anoroc** [comment]
Thanks for the input @icyJoseph 

The majority of the site is pre-rendered. We do, however, have pages with dynamic content where I'm using ISR given the number of pages that would be required to create at build time. 

FCP typically measures below 2s, with a few spikes going up to to wild numbers. I assume this is due to visitors searching on machines with poor performance/connection speeds or mobile in spotty service areas. 

LCP is typically within a healthy range, but still suffers from random spikes going to silly numbers that just don't seem to make a whole lot of sense (These occur more infrequently). 

CLS is close to, if not always 0 as of late. 

FID is typically in the single digits (last check shows an average of 7 ms)
